regression/goto-analyzer-simplify/simplify-multiply-by-zero/test.desc

```diff
@@ -0,0 +1,11 @@
+CORE
+main.c
+"--variable"
+^SIGNAL=0$
+^EXIT=6$
+main#return_value = 0;
+--
+--
+Tests that a multiplication
+of variable*variable can be simplified
+if one of the variables can be evaluated to 0.
```

Comment:
As above: missing newline. (Or at least that's how I read github's diff with the red marker at the end.)